File: bank-service/src/main/java/rs/edu/raf/repository/CardIssuerRepository.java
package rs.edu.raf.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import rs.edu.raf.model.card.CardIssuer;

@Repository
public interface CardIssuerRepository extends JpaRepository<CardIssuer, Long> {
}

File: bank-service-kotlin/src/main/kotlin/org/example/bankservicekotlin/repository/CardIssuerRepository.kt
package org.example.bankservicekotlin.repository

import org.example.bankservicekotlin.model.card.CardIssuer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface CardIssuerRepository : JpaRepository<CardIssuer, Long> {
}